assets/scripts/brewery.js

```diff
@@ -2,37 +2,67 @@
 //var cityName = "miami";
 var breweryObjects ;
 
-
-
-
 function randomByCity (city) {
   var cityName = city;
   var beerByCityURL = "https://api.openbrewerydb.org/breweries?by_city=" + cityName;
   makeAjaxRequest(beerByCityURL, renderResults );
 };
 
-
 // REFER TO https://api.eventful.com/docs/events/search
 function beerEvents (city) {
   var cityName = city;
   var eventFulURL = "https://cors-anywhere.herokuapp.com/http://api.eventful.com/json/events/search?keywords=beer&app_key=dSgcPNSbCFTGkjzg&location=" + cityName;
   makeAjaxRequest(eventFulURL, consolelogResults );
 }
 
-
 function renderResults (result) {
   // store the results globally to be used elsewhere and local storage if needed.
   breweryObjects = result;
+  $('tbody').empty();
   for (i = 0; i < breweryObjects.length; i++) {
     
-      var tableRow = $("<tr>");
-      tableRow.append('<td style="width: 10%; text-align: center" class="col"><button class="detailBtn Btn" type="button" data-toggle="modal" data-index=' + [i] + ' data-target="#breweryDetail"><i class="fas fa-info-circle"></i></button><button class="editBtn Btn d-flex" type="button" data-toggle="modal" data-index=' + [i] + '><i class="far fa-edit"></i></button></td>');
+      var editBtn = $("<a>").attr({
+        class:"editBtn Btn d-inline text-muted",
+        href:"#",
+        type:"button",
+        'data-toggle': "modal",
+        'data-target': "#breweryDetail"
+        });
+      editBtn.data({
+          'index': i
+        });
+      var editIcon = $('<i>').addClass("far fa-edit p-1");
+      editBtn.append(editIcon);
+      
+      var infoBtn = $("<a>").attr({
+        class:"detailBtn Btn d-inline text-muted",
+        href:"#",
+        type:"button",
+        'data-toggle': "modal",
+        'data-target': "#breweryDetail"
+        });
+      var infoIcon = $('<i>').addClass("fas fa-info-circle");
+
+      infoBtn.data({
+          'index': i
+        });
+      infoBtn.append(infoIcon);
+
+      var buttonCol = $('<td>').attr({
+        style:"width:20%; text-align: center",
+        class:"col align-center "
+      });
+      buttonCol.append(infoBtn, editBtn)
+      var tableRow = $("<tr>").attr("data-row", i);
+      tableRow.addClass("breweryRow");
+ 
+      tableRow.append(buttonCol);
 
       tableRow.append('<td style="width: 50%; text-align: left" class="col">' + breweryObjects[i].name + '</td>');
       var phone = formatPhone(breweryObjects[i].phone);
       tableRow.append('<td style="width: 40%; text-align: center" class="col">' + phone + '</td>');
       $('tbody').append(tableRow);
```

Comment:
Broke up the code to create the buttons within the table. More lines... but easier to read and build out.